### ISSUE TITLE
Attempted import error: 'read' is not exported from './io/reader'.

### ISSUE THREAD

**Issue by halflings** [opensource9ja/danfojs#215]
**Describe the bug**
Getting this error when import danfojs (in typescript, for a React application):
`import * as dfd from "danfojs/src/index";`

Note that typescript complains about the lack of types as well:
```
Could not find a declaration file for module 'danfojs/src/index'. '//wsl$/Ubuntu/home/kachkach/workspace/mlconsole/public/node_modules/danfojs/src/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/danfojs` if it exists or add a new declaration (.d.ts) file containing `declare module 'danfojs/src/index';`
```
**To Reproduce**
1. Import danfo: `import * as dfd from "danfojs/src/index";`
2. Try to create a dataframe: `new dfd.DataFrame(data, {columns: columns})`
3. Get this error: 
```
./node_modules/danfojs/src/index.js
Attempted import error: 'read' is not exported from './io/reader'.
```

**Expected behavior**
Danfo can be successfully imported.

**Comment by polvoazul** [comment]
it seems like this commit 2a0614380edc74dc3cbe481f7c5674d182af92dd removed the method but forgot to update the types

**Comment by risenW** [comment]
@halflings Thanks for the update. I totally missed that.

FIXED in new release [(v0.2.7)](https://www.npmjs.com/package/danfojs)
